js/main.js

```diff
@@ -208,7 +210,7 @@ var attentionScum = function(vocabulary) {
 var buildQuote = function(vocabulary) {
     var quote = '';
 
-    if (Math.random() < 0.85) {
+    if (_.random(9) < 8) { // 80% chance
```

Comment:
Makes perfect sense — striking that balance between readable and overly verbose is tricky. I'll defer to your preference here, and push an update shortly.